### ISSUE TITLE
Messages may contain multiple parts

### ISSUE THREAD

**Issue by StevenMaude** [StevenMaude/go-mailin8#3]
One example I've seen:

```
    "parts": [
        {
            "headers": {
                "content-transfer-encoding": "8bit",
                "content-type": "text/plain; charset = \"utf-8\""
            },
            "body": "..."
        },
        {
            "headers": {
                "content-transfer-encoding": "8bit",
                "content-type": "text/html; charset = \"utf-8\""
            },
            "body": "..."
        }
```

**Comment by StevenMaude** [comment]
The first part body only contains `<br>`, while the second part body contains the same text but with what looks like full markup, including the `<a>` that's needed in this case (and missing from the first part).

Including the markup would be messy (see #4) but would at least include the URL(s) of interest.

**Comment by StevenMaude** [comment]
Hopefully fixed by #5.
